src/App.js

```diff
@@ -6,12 +6,13 @@ import Button from './components/Button/Button';
 import Card from './components/Card/Card';
 
 class App extends Component {
-
-  cardImg = {
-    url : "http://placehold.it/200x32",
-    alt : "Placeholder"
+  constructor() {
+    super();
+    this.image = {
```

Comment:
Sorry to say this now, only just thought of it, but if you're only going to use this 64x64 image for the cards, I'd move this from being an instance variable and just put it at the top of the `render` method as a normal const... But this works too, so up to you!